source/blog/2019-05-11-rspec-rails-4-0-has-been-released.md

```diff
@@ -0,0 +1,181 @@
+---
+title: RSpec Rails 4.0 has been released!
+author: Benoit Tigeot, Jon Rowe and Phil Pirozhkov
+---
+
+RSpec Rails 4.0 has been released recently! Given our commitment to
+[semantic versioning](http://semver.org/), this update comes with 
+breaking changes for people that are using Rails below 4.2. If we 
```

Comment:
Maybe put this in a separate paragraph to emphasize that bugs aren't an integral part of the new release? :D

> As usual, if you notice any newly introduced bugs, please let us know, and we'll release a fix for it ASAP.